### PR TITLE
feat: enable codecarbon by default

### PR DESCRIPTION
**Pull request by Samoed**
This PR enables code carbon by default. Closes https://github.com/embeddings-benchmark/mteb/issues/1427

## Checklist

- [X] Run tests locally to make sure nothing is broken using `make test`. 
- [X] Run the formatter to format the code using `make lint`.

### CODE REVIEW

File: mteb/cli.py [hunk 1]
```diff
@@ -265,8 +265,8 @@ def add_run_parser(subparsers) -> None:
     parser.add_argument(
         "--co2_tracker",
         type=bool,
-        default=False,
-        help="Enable CO₂ tracker, disabled by default",
+        default=True,
+        help="Enable CO₂ tracker, enabled by default",
     )
```
Comment (isaac-chung) [inline mteb/cli.py:270]: Now that this option is true by default, we might want to offer a `disable_co2_tracker` instead.

**Review by isaac-chung — COMMENTED** [review]
Nice, thanks! Let's get the tests passing as well.

**Comment by Samoed** [comment]
Locally, the tests are passing, and I’m not sure why they're failing in the CI environment.

**Comment by Samoed** [comment]
It seems that the emission tracker fails due to multiple workers, as it works fine when I run it without `-n auto`.

**Comment by KennethEnevoldsen** [comment]
Since this is technically breaking I will move it to v2.0.0

**Comment by isaac-chung** [comment]
Based one [this issue](https://github.com/mlco2/codecarbon/issues/681), it seems like we can* allow multiple runs, but the tracking will be somewhat skewed. It won't matter in the tests, but will for sure affect real runs if multiple runs are done on the same machine. I think we would also add a line in README and/or a warning to let users know.

**Comment by KennethEnevoldsen** [comment]
Sounds like this can be merged into v2.0.0?

**Comment by isaac-chung** [comment]
Yes, just missing a line in README and/or a warning in the comment above.

**Comment by Samoed** [comment]
@isaac-chung I added logging message

**Pull request merged**